spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4233,6 +4234,27 @@ def __in_block_comment(cursor):
         cursor3.endEditBlock()
         return True
 
+    def create_new_cell(self):
+        firstline = '#%% Start Cell \r\n'
+        endline = '\r\n'
+        cursor = self.textCursor()
+        if self.has_selected_text():
+            self.extend_selection_to_complete_lines()
+            start_pos, end_pos = cursor.selectionStart(), cursor.selectionEnd()
+            endline = '\r\n#%% End Cell'
+        else:
+            start_pos = end_pos = cursor.position()
+        cursor.beginEditBlock()
+        cursor.setPosition(start_pos)
+        cursor.movePosition(QTextCursor.StartOfBlock)
```

Comment:
```suggestion
```

These two instructions don't seem necessary because they are repeated below. However, I'm not really sure about it, so @jsbautista please check that before applying my suggestion.